portfolio.js

```diff
@@ -22,3 +22,234 @@ for (let i = 0; i < navLinks.length; i += 1) {
+    technologies: ['Ruby on rails', 'css', 'javascript', 'html'],
+    link_to_Source: '#',
+    link_to_live_version: '#',
+    mobile_popup_technologies: ['Ruby on rail', 'css', 'Javascript'],
+    desktop_popup_technologies: ['Codekit', 'Github', 'Javascript', 'Bootstrap', 'Terminal', 'Codepen'],
+  },
+
+];
+
+const projectsContainer = document.getElementById('projects-container');
+
+function showModal() {
+  if (!!document.querySelector('.popup-wrapper') === true) {
+    document.querySelector('.popup-wrapper').remove();
+  }
+  document.querySelector('.popup-wrapper').style.display = 'block';
+}
+
+// dynamically create the project section
+
+for (let i = 0; i < projectsDetails.length; i += 1) {
+  const projectFile = document.createElement('div');
+  projectFile.classList.add('project-file');
+
+  const projectImage = document.createElement('div');
+  projectImage.classList.add('project-image');
+
+  const imagePlaceholder = document.createElement('div');
+  imagePlaceholder.classList.add('image-placeholder');
+
+  projectImage.appendChild(imagePlaceholder);
+
+  projectFile.appendChild(projectImage);
+
+  const projectText = document.createElement('div');
+  projectText.classList.add('project-text');
+
+  const projectTitle = document.createElement('h3');
+  projectTitle.classList.add('project-title');
+  projectTitle.textContent = projectsDetails[i].title;
+
+  projectText.appendChild(projectTitle);
+
+  const projectLangs = document.createElement('div');
+  projectLangs.classList.add('project-langs');
+
+  const langList = document.createElement('ul');
+  langList.classList.add('lang-list');
+
+  for (let j = 0; j < projectsDetails[i].technologies.length; j += 1) {
+    const lang = document.createElement('li');
+    lang.textContent = projectsDetails[i].technologies[j];
+    langList.appendChild(lang);
+  }
+
+  projectLangs.appendChild(langList);
+
+  projectText.appendChild(projectLangs);
+
+  const projectButton = document.createElement('div');
+  projectButton.classList.add('project-button');
+
+  const seeProjectButton = document.createElement('button');
+  seeProjectButton.classList.add('seeModal');
+  seeProjectButton.type = 'button';
+  seeProjectButton.textContent = 'See Project';
+
+  projectButton.appendChild(seeProjectButton);
+
+  projectText.appendChild(projectButton);
+
+  projectFile.appendChild(projectText);
+
+  projectsContainer.appendChild(projectFile);
+
+  seeProjectButton.addEventListener('click', showModal);
+}
+
+function closeModal() {
+  document.querySelector('.popup-wrapper').style.display = 'none';
+}
+
+// dynamically create popup elements
+
+for (let i = 0; i < projectsDetails.length; i += 1) {
+  const popupWrapper = document.createElement('div');
+  popupWrapper.classList.add('popup-wrapper');
```

Comment:
Well done on implementing the changes requested by the previous reviewer 👏🏾

[OPTIONAL] - To improve your UI you can add either a scroll or flexible height on your popup so that the description text does not overlap outside of the modal container.

![image](https://user-images.githubusercontent.com/38088818/168275195-c00cf760-5dc0-4b18-9fdc-2429947c2e8a.png)